CONTRIBUTING.md

```diff
@@ -366,3 +366,79 @@ duration = float_or_none(video.get('durationMs'), scale=1000)
 view_count = int_or_none(video.get('views'))
 ```
 
+### Inline values
+
+ Always inline values.
+
+#### Example
+
+A good rule of thumb is not to create variables that will only be used once.
+
+Correct:
+
+```python
+return self.playlist_result(
+    [self._parse_brightcove_metadata(vid, vid.get('id'), headers)
+	for vid in json_data.get('videos', []) if vid.get('id')],
+    json_data.get('id'), json_data.get('name'),
+    json_data.get('description'))
+```
+
+Incorrect:
+
+```python
+playlist_vids = json_data.get('videos', [])
+playlist_id = json_data.get('id')
+playlist_title = json_data.get('name')
+playlist_description = json_data.get('description')
```

Comment:
Examples should be simple and concise. Here you have the same pattern occurring four times that is superfluous.

Reply:
Understood. I changed it to use one variable assignment instead of four.